app/services/oneoffs/npq/data_cleanup/fix_applications_in_ecf_not_in_npq.rb

```diff
@@ -0,0 +1,90 @@
+# frozen_string_literal: true
+
+module Oneoffs::NPQ::DataCleanup
+  class FixApplicationsInECFNotInNPQ
+    attr_reader :npq_application_ids
+
+    def initialize(npq_application_ids:)
+      @npq_application_ids = npq_application_ids
+    end
+
+    def run!(dry_run: true)
```

Comment:
I feel like a broken record today, but this is a pretty big method that might benefit from extracting into smaller methods 🙂